doc/nrf/libraries/bin/lwm2m_carrier/app_integration.rst

```diff
@@ -142,6 +142,8 @@ Following are the various LwM2M carrier library events:
 
     * :c:macro:`LWM2M_CARRIER_DEFERRED_SERVER_REGISTRATION` - The server registration has not completed, and the server does not recognize the connecting device. If this event persists, contact the carrier.
 
+    * :c:macro:`LWM2M_CARRIER_DEFERRED_SERVICE_UNAVAILABLE` - The server is in maintenance mode. The client will not attempt to set up a session for the next 4 hours.
```

Comment:
The delay is indicated in `lwm2m_carrier_event_deferred_t.timeout`, not hard coded to 4 hours.

Reply:
Ok, then I suppose it's not necessary to mention the timeout here